servicetalk-loadbalancer/src/main/java/io/servicetalk/loadbalancer/RoundRobinLoadBalancer.java

```diff
@@ -86,6 +88,9 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(RoundRobinLoadBalancer.class);
     private static final List<?> CLOSED_LIST = new ArrayList<>(0);
+    private static final Object[] CLOSED_ARRAY = new Object[0];
```

Comment:
Consider updating copyright year for this class

Reply:
done